### PR TITLE
fix(files): Fix reference picker not working

### PR DESCRIPTION
**Pull request by susnux**
This resolves https://github.com/nextcloud/server/issues/40866 for stable 27.

The nodes are an array of all selected nodes even if multiselect is false, so we need to use the first element.

### CODE REVIEW

File: apps/files/src/views/FileReferencePickerElement.vue [hunk 1]
```diff
@@ -52,7 +52,7 @@ export default {
 				{
 					label: t('files', 'Choose'),
 					type: 'primary',
-					callback: (node) => this.submit(node.fileid),
+					callback: (nodes) => { console.warn('got', nodes); this.submit(nodes[0].fileid) },
```
Comment (juliushaertl) [inline apps/files/src/views/FileReferencePickerElement.vue:55]:
```suggestion
					callback: (nodes) => { this.submit(nodes[0].fileid) },
```

or log at debug level but with a more clear message?

Comment (emoral435) [reply apps/files/src/views/FileReferencePickerElement.vue:55]: Agreed, just change the log with a clearer message and then it LGTM :)

Comment (susnux) [reply apps/files/src/views/FileReferencePickerElement.vue]: Done

**Review by juliusknorr — APPROVED** [review]

**Review by artonge — APPROVED** [review]

**Review by skjnldsv — APPROVED** [review]

**Review by AndyScherzinger — APPROVED** [review]

**Review by emoral435 — APPROVED** [review]

**Pull request merged**